src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CrlCache.cs

```diff
@@ -237,13 +237,17 @@ private static string GetCdpUrl(SafeX509Handle cert)
                         }
                     }
                 }
-
-                return null;
+            }
```

Comment:
should we also try to read with BER rules?

Reply:
No, certificates are required to be DER.  So an extension that's legal-BER-but-not-legal-DER is as broken as one that is has non-nested lengths.